app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -45,13 +48,28 @@ export default function NoteForm({ onClose, refetch, note, isEdit }) {
               label="Description"
               name="description"
               className="w-full flex-grow-0"
-              rows={8}
+              rows={4}
+              required
+            />
+            <Select
+              className="w-full flex-grow-0"
+              label="Assigned Contact"
+              name="assignedContact"
+              options={ASSIGNED_CONTACT}
+              required
```

Comment:
Implicitly true values should always be written as the first set of attrs in the attr list.